### DISCUSSION TITLE
Comment on Expanded, Un-Changed Sections in PR

### DISCUSSION THREAD

**Discussion by dietza** [community/community#13029] [Pull Requests]
When reviewing a PR, sections of code are collapsed to hide unchanged lines while the changes within a file are highlighted.
I can expand the hidden sections to view the unchanged code in the file, but -
if I want to make a comment / ask a question on one of the previously hidden lines, I am unable to do so.
I would expect that the same functionality and flow would apply here as to the exposed sections of changes, and would just open the comment drawer on the left (prev version) side of my screen.

___

Apologies for redundancy - I see that this is addressed here: https://github.com/github/feedback/discussions/4452 , https://github.com/github/feedback/discussions/9099

**Comment by netaalon** [comment]
Was also addressed in [this unofficial GitHub repo](https://github.com/isaacs/github/issues/1655). Seems to me to be super useful!

**Reply by netaalon** [reply]
Took a look in #4452 and was excited to see this is in the plans for the next quarter :)
